WebRoot/static/dhtml/main.js

```diff
@@ -1,160 +1,131 @@
-// ECMAScript 5
-"use strict";
+// Global Variables
+let url = window.location.href;
+let body = document.body.style;
+let icon = document.getElementById("menu-icon");
+let winWidth = window.innerWidth;
```

Comment:
😱 No respect for this poor ES5... Please do something!

Reply:
hahahaha love your comments